src/System.Runtime/src/System/Reflection/AssemblyNameProxy.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+**
+**
+**
+**
+**
+** Purpose: Remotable version the AssemblyName
+**
+**
+===========================================================*/
+namespace System.Reflection {
+    using System;
+    using System.Runtime.Versioning;
+
+    [System.Runtime.InteropServices.ComVisible(true)]
+    public class AssemblyNameProxy : MarshalByRefObject
```

Comment:
The runtime method was excluded from the VM implementation with `ifdef`s. I believe I could expose this method and a few more Reflection APIs with similar situation, but at the time I was not aware of the reason why the implementation didn't exist for those runtime methods.
I will use `AssemblyLoadContext `for now and then I can go over the list of not yet exposed Reflection APIs in another iteration to see if I can expose any more. Please let me know if this sounds good to you.